server/templates/explore.html

```diff
@@ -25,6 +25,7 @@
  {% block head %}
  <link rel="stylesheet" href={{url_for('static', filename='css/explore.min.css', t=config['VERSION'])}} >
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
```

Comment:
Looks like this adds a pretty large asset (network panel shows 3.1mb) - is there a way to grab fewer icons?